examples/cc26xx/cc26xx-web-demo/cc26xx-web-demo.c

```diff
@@ -966,6 +996,54 @@ PROCESS_THREAD(cc26xx_web_demo_process, ev, data)
 
   PROCESS_END();
 }
+
+PROCESS_THREAD(adc_process, ev, data)
+{
+  PROCESS_BEGIN();
+  static struct etimer et_adc;
+  etimer_set(&et_adc, CLOCK_SECOND * 5);
+  while(1) {
+    PROCESS_WAIT_EVENT_UNTIL(etimer_expired(&et_adc));
+    /* intialisation of ADC */
+    ti_lib_aon_wuc_aux_wakeup_event(AONWUC_AUX_WAKEUP);
+    while(!(ti_lib_aon_wuc_power_status_get() & AONWUC_AUX_POWER_ON)) {
+    }
+
+    /* Enable clock for ADC digital and analog interface (not currently enabled in driver) */
+    /* Enable clocks */
+    ti_lib_aux_wuc_clock_enable(AUX_WUC_ADI_CLOCK | AUX_WUC_ANAIF_CLOCK | AUX_WUC_SMPH_CLOCK);
+    while(ti_lib_aux_wuc_clock_status(AUX_WUC_ADI_CLOCK | AUX_WUC_ANAIF_CLOCK | AUX_WUC_SMPH_CLOCK) != AUX_WUC_CLOCK_READY) {
+    }
+    /* printf("clock selected\r\n"); */
```

Comment:
I suggest removing unused printfs (and the windows-style of `\r\n` isn't used elsewhere, the `\n` is the line ending used everywhere else).